source/Cosmos.IL2CPU/IL/Ldelem_Ref.cs

```diff
@@ -25,7 +25,29 @@ public static void Assemble(Assembler aAssembler, uint aElementSize, bool isSign
     {
       //  stack     = index
       //  stack + 2 = array
-      DoNullReferenceCheck(aAssembler, debugEnabled, 8);
+      // null reference check is done in ldlen
```

Comment:
The pattern we use for loading references/pointers is to do the null check at the beginning.

Reply:
Ok changed it